### DISCUSSION TITLE
please delete

### DISCUSSION THREAD

**Discussion by ghost** [NVIDIA/open-gpu-kernel-modules#306] [Polls]

**Comment by yoshimo** [comment]
While the sourcecode , together with the signing key, would be a great step, i think it would be useful enough already to get the binary files out under a permissive license to people can reclock their devices and get the hardware the proper support that it needs.

**Comment by YusufKhan-gamedev** [comment]
Liberated source code means nothing, we really just need gpg keys.

**Reply by YusufKhan-gamedev** [reply]
Im pretty sure that with gpg keys we could get free PM and fan control firmware *easily(not easy)* by extending the already existing libre Falcon Firmware.

**Comment by throwaway1037** [comment]
This is a duplicate of #292.
Also, merely releasing the source code is not enough to respect freedom.
The source code must be released under a free software license, not depend on any proprietary components, and the final binary generated from the source code must be able to be signed by the end-user, so people can actually load modified versions onto their own cards.

**Comment by aaronp24** [comment]
Let's keep this discussion in one of the existing topics. Closing this one.
